Profile Folder/chrome/locale/zh-CN/dtd/gsettings.dtd

```diff
@@ -108,9 +108,9 @@
 <!ENTITY gsettings.appearance                           "外观">
 <!ENTITY gsettings.openChrThemesFolder				    "打开 Chromium 主题文件夹">
 <!ENTITY gsettings.refreshList					        "刷新列表">
-<!ENTITY gsettings.filename								"[!!! Fïℓèñá₥è: !!!]">
+<!ENTITY gsettings.filename								"文件名：">
 <!ENTITY gsettings.viewStorePage						"[!!! Vïèω §ƭôřè ƥáϱè !!!]">
```

Comment:
`viewStorePage` is the button that will take you to a theme's page.
Examples:
![image](https://github.com/user-attachments/assets/fd579ade-3b46-4129-837c-ac6193fb004a)
https://addons.mozilla.org/en-US/firefox/addon/chiaki-nanami-blue/
![image](https://github.com/user-attachments/assets/55ab6d9d-7b0e-4e42-bacf-588140097f9b)
https://chromewebstore.google.com/detail/angry-birds/coaaadiiedpfjmpkampmbbfbnolpchik